### PR TITLE
Change import format from .csv to .tsv

### PR DESCRIPTION
**Pull request by F3PiX**
Maria reported problems with uploading a .csv file in production., from a Google Spreadsheet export.
Google doesn't allow to override the column separator, other than choosing 'csv' or 'tsv'.
After changing the `col_sep` option to `','` , for Maria the date format with comma's inside ('Aug 7, 2017') did not cause problems, but on my computer, the comma inside was read as an extra separator. 

The safest solution is to use the tab separated values content type.

@carpodaster Do you agree?
(And look how easy the `content_type` changes were!  🙏  )

### CODE REVIEW

File: spec/fixtures/files/test.tsv [hunk 1]
```diff
@@ -0,0 +1,7 @@
+UID/tName/tStart date/tEnd date/tCity/tCountry/tRegion/tWebsite/tNotes
```
Comment (carpodaster) [inline spec/fixtures/files/test.tsv]: Is this what an actual Google export looks like? The  `/t` isn't a tab escape. `\t` would be. That's why I am curious.

Comment (F3PiX) [inline spec/fixtures/files/test.tsv]: 🙈

**Comment by F3PiX** [comment]
I didn't find that we can tell Google to do that, but I am not very into Google Docs. On Maria's computer it does surround with quotes, on mine it doesn't. 
Either way, it would require more 'instructions' on what the imported file should look like. And that I don't like much.

**Comment by F3PiX** [comment]
I searched about the Google settings. Google doesn't allow to set export settings, but it does allow import settings. So, that requires an extra import action into a new file, and than we get:
```
UID,Name,Start date,End date,City,Country,Region,Website,Notes
2017001,Deccan RubyConf 2017,"Aug 12, 2017","Aug 12, 2017",Gent,Belgium,Europe,http://www.deccanrubyconf.org/,
```
I think that is a bit unpractical and error prone. 
Are there objections against .tsv ?
@carpodaster

**Comment by F3PiX** [comment]
Closing; see #800

**Pull request closed without merging**